packages/common/src/workflow-options.ts

```diff
@@ -7,11 +7,14 @@ import { checkExtends } from './type-helpers';
 // Avoid importing the proto implementation to reduce workflow bundle size
 // Copied from temporal.api.enums.v1.WorkflowIdReusePolicy
 /**
+ * Defines what happens when trying to start a Workflow with the same ID as a *Closed* Workflow.
+ *
+ * See {@link WorkflowOptions.workflowIdConflictPolicy} for what happens when trying to start a Workflow with the same ID as a *Running* Workflow.
```

Comment:
Please break long comment lines. prettier won't do it for you.

The project is not consistent on this regard (because of no tooling to automate that), but I personally strive to get good readability at 100 columns, allowing occasional overflows… Whatever that keeps the code comfortable to read. Note however that you can't never break a line in the middle of a `{@link ...}` and similar tags; the parser won't handle that. So having extra long lines with those tags trailing on the right side is ok.

Reply:
I wonder if we could use this https://prettier.io/docs/en/options.html#print-width but that could break all the`{@link}`...